ambry-api/src/main/java/com.github.ambry/config/StoreConfig.java

```diff
@@ -112,6 +119,8 @@ public StoreConfig(VerifiableProperties verifiableProperties) {
     storeEnableHardDelete = verifiableProperties.getBoolean("store.enable.hard.delete", false);
     storeSegmentSizeInBytes =
         verifiableProperties.getLongInRange("store.segment.size.in.bytes", Long.MAX_VALUE, 1, Long.MAX_VALUE);
+    storeHardDeleterSleepTimeWhenCaughtUpInMs =
```

Comment:
Sorry, I was able to fix this. In tests since we use MockTime, once pause() is called, advancing the MockTime will wake up the hard deleter thread (if awaiting) and then we do our usual validation for thread's state.